interfaces/builtin/x11.go

```diff
@@ -83,6 +83,11 @@ unix (bind, listen, accept)
 # For Xorg to detect screens
 /sys/devices/pci**/boot_vga r,
 /sys/devices/pci**/resources r,
+
+# Allow writing an Xwayland Xauth file
+# (see https://gitlab.gnome.org/GNOME/mutter/merge_requests/626)
+/run/user/[0-9]*/.mutter-Xwaylandauth.* rw,
+/run/user/[0-9]*/mutter/Xauthority rw,
```

Comment:
Given #9952, should we add a rule for kwin too?
```
/run/user/[0-9]*/xauth_* rw,
```

Reply:
I removed adding any rules from this PR to x11 slot as per @jhenstridge's recommendation